src/utils/index.ts

```diff
@@ -107,10 +107,19 @@ const autodetectableProviders = {
   cleavr: "cleavr",
 };
 
+const autodetectableStaticProviders = {
+  vercel: "vercel-static",
+};
+
 export function detectTarget() {
   return autodetectableProviders[provider];
 }
 
+// TODO: update when https://github.com/unjs/nitro/pull/1072 merges
+export function detectStaticTarget() {
```

Comment:
PR mergeed, also refactor, we might support `{ static }?` for `detectTarget` itself to pas conditions like this without introducing new one

Reply:
I mean not all options but detect conditions (practically merging two utils into one)